actix-web-codegen/tests/trybuild.rs

```diff
@@ -12,6 +12,8 @@ fn compile_macros() {
     t.compile_fail("tests/trybuild/route-unexpected-method-fail.rs");
     t.compile_fail("tests/trybuild/route-malformed-path-fail.rs");
 
+    t.pass("tests/trybuild/routes-ok.rs");
```

Comment:
Maybe add some negative tests? E.g. with `routes` macro without any actual routes supplied?

Reply:
I think in that case it would just do nothing, but an error would make sense.